### PR TITLE
Quill typings - remove duplicate interface, add static keyword to three methods

### PR DESCRIPTION
**Pull request by ericeslinger**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Context for update: https://quilljs.com/docs/api/#register 

There are two changes here, one of which may be unnecessary, I suppose. The important change I made was to flag `import`, `register`, `debug`, and `find` as static methods on the class. The second change I made was to eliminate the duplicative interface Quill - having a class for it should be sufficient (they were identical save for the EventEmitter extension).

@sumitkm @guillaume-ro-fr

### CODE REVIEW

File: types/quill/index.d.ts [hunk 1]
```diff
@@ -128,71 +128,7 @@ export interface EventEmitter {
...
-    find(domNode: Node, bubble?: boolean): Quill | any;
-    getIndex(blot: any): number;
-    getLeaf(index: number): any;
-    getLine(index: number): [any, number];
-    getLines(index?: number, length?: number): any[];
-    getLines(range: RangeStatic): any[];
-}
-
-export class Quill implements Quill {
+export class Quill implements Quill, EventEmitter {
```
Comment (guillaume-ro-fr) [inline types/quill/index.d.ts]: @ericeslinger : What is the `Quill` interface ? (Not defined ?)

**Review by guillaume-ro-fr — COMMENTED** [review]
Quill interface

File: types/quill/index.d.ts (hunk 1 above)
Comment (ericeslinger) [inline types/quill/index.d.ts]: woops, good catch. fixing.

**Comment by guillaume-ro-fr** [comment]
👍

**Comment by guillaume-ro-fr** [comment]
@dt-bot 👍

**Pull request merged**